internal/servers/controller/handlers/authmethods/authmethod_service.go

```diff
@@ -63,29 +65,30 @@ var (
 
```

Comment:
Out of scope of this PR, but why not wrap these package vars (IdActions, CollectionActions, collectionTypeMap) in funcs?  I'm happy to take this work on.  pkg maps just seem to be a data race or panic waiting to happen in the future.

Reply:
We could. Honestly they should be const, but you can't declare maps as const :-/ Won't do it in this PR though.